proposals/lambda-method-group-defaults.md

```diff
@@ -18,9 +22,16 @@ var addWithDefault = addWithDefaultMethod;
 addWithDefault(); // 3
 addWithDefault(5) // 6
 
+var counter = countMethod;
```

Comment:
>counter [](http://example.com/codeflow?start=4&length=7)

Minor: It looks like we switch between `var count = ...` and `var counter = ...` throughout the document. Unless there is a reason for having two distinct names, consider choosing one.

Reply:
Good catch, thanks. 0661f1fb0b893c301f2d8255fcd1fd08d1a04939